nextflow.config

```
@@ -18,15 +18,15 @@ params {
     igenomes_base              = 's3://ngi-igenomes/igenomes'
     igenomes_ignore            = true
 
-    // Macrel
-    run_amp_macrel             = false
+    // AMP options
+    run_amp_screening           = false
+    skip_amp_macrel             = false
 
-    // fARGene options
-    run_arg_fargene            = false
+    // ARG options
+    run_arg_screening           = false
+    skip_arg_fargene            = false
     fargene_hmm_model          = 'class_a'
-
-    // DeepARG options
-    run_arg_deeparg                     = false
+    skip_arg_deeparg                     = false
     deeparg_data                        = null
     deeparg_model                       = 'SS'
     deeparg_type                        = 'nucl'
```

A suggestion would be to group tool options under subworkflow prefix.

```suggestion
    arg_deeparg_data                        = null
    arg_deeparg_model                       = 'SS'
    arg_deeparg_type                        = 'nucl'
```

---

Agreed, good idea

---

Done in https://github.com/nf-core/funcscan/pull/46/commits/940a16d37901ab89078dd6f2d130c8dbb59c9f8f